src/tests/apitests.ts

```diff
@@ -189,7 +189,7 @@ describe("Laputin API", function() {
         rimraf.sync(archivePath);
         fs.mkdirSync(archivePath);
 
-        var laputin = compose(archivePath, new LaputinConfiguration(1234, "accurate"));
+        var laputin = compose(archivePath, new LaputinConfiguration(null));
```

Comment:
Instead of null you should pass { "identification": "accurate" } here to fix failing tests.